zerver/openapi/zulip.yaml

```diff
@@ -11104,6 +11104,12 @@ paths:
                     only options to disable sending typing notifications.
                   type: boolean
                   example: true
+                automatically_detect_time_zone:
+                  description: |
+                    Option to detect browser time zone and offer to update the user
+                    setting time zone with it.
```

Comment:
Please read our "new application feature" tutorial section on API changes and documenting this; or use `git log` on this file to see a lot of examples of doing it properly. You need a `**Changes**` entry, etc. And it needs to be very carefully written prose aligning with existing patterns.

Reply:
Documentation is new territory for me and I had planned to address it once the core functionality is in a good state. Thanks for the suggestion, I'll change it

Reply:
I have updated the documentation and the changelog